spec/spec-helper.coffee

```diff
@@ -17,13 +18,18 @@ getEditorElement = (callback) ->
     element = document.createElement("atom-text-editor")
     element.setModel(textEditor)
     element.classList.add('vim-mode')
-    element.vimState = new VimState(element, new StatusBarManager, new GlobalVimState)
+    statusBarManager = existingVimState?.statusBarManager or new StatusBarManager
+    globalVimState = existingVimState?.globalVimState or new GlobalVimState
+    element.vimState = new VimState(element, statusBarManager, globalVimState)
 
     element.addEventListener "keydown", (e) ->
       atom.keymaps.handleKeyboardEvent(e)
 
     callback(element)
 
+# this function puts the existingElement parameter first
+getOtherEditorElement = (existingElement, callback) -> getEditorElement(callback, existingElement)
+
```

Comment:
I think the best way of sharing the `GlobalVimState` between the two editors would be to store it as a local variable in this file (`globalVimState = null` at the top of the file) and initialize it in this file's `beforeEach` block. That way, we could create other editors without needing a reference to the first editor.